modules/images-other-jenkins-agent-images.adoc

```diff
@@ -12,27 +12,12 @@ Jenkins images are available through the Red Hat Registry:
 
 [source,terminal]
 ----
-$ docker pull registry.redhat.io/openshift4/ose-jenkins:<v4.5.0>
+$ docker pull registry.redhat.io/ocp-tools-4/ose-jenkins:<image_tag>
 ----
 
 [source,terminal]
 ----
-$ docker pull registry.redhat.io/openshift4/jenkins-agent-nodejs-10-rhel7:<v4.5.0>
-----
-
-[source,terminal]
-----
-$ docker pull registry.redhat.io/openshift4/jenkins-agent-nodejs-12-rhel7:<v4.5.0>
-----
-
-[source,terminal]
-----
-$ docker pull registry.redhat.io/openshift4/ose-jenkins-agent-maven:<v4.5.0>
-----
-
-[source,terminal]
-----
-$ docker pull registry.redhat.io/openshift4/ose-jenkins-agent-base:<v4.5.0>
+$ docker pull registry.redhat.io/ocp-tools-4/jenkins-agent-base:<image_tag>
```

Comment:
the image has `rhel8` in the name
https://catalog.redhat.com/software/containers/ocp-tools-4/jenkins-agent-base-rhel8/6241e3457847116cf8577aea